### PR TITLE
Nanobot fluid, Kyphotorin fix, and more

### PR DESCRIPTION
**Pull request by ThePainkiller**
## Because if I don't code shit for medical no one will

- Introducing Nanobot Fluid, a chemical that heals exclusively the external and internal wounds of prosthetics. Works only on actual humans with prosthetics and not on FBP's as they have no bloodstream to act upon. Obtain by mixing equal parts Cronexidone, Aluminum and Iron, with 0.1 parts of Raw Uncapped Nanites in the prescence of 5u Plasma, at low enough temperatures.
- Kyphotorin now works on the dead, being able to regrow heads and limbs while used on a cryopod
- Refactors "clonexadone" into "cronexidone" proper, Cronexidone now actually heals around 20% more brute and burn than Cryoxadone, making its description no longer lies, it also fixes internal organs very slightly
- Removes all references to "clone" damage where possible, replaces them with "cellular damage/corruption" where applicable
- Fixes certain toolmods/gunmods being slapped into armors and RIG modules (you can still apply beeswax and anti-stain coating to them)
- Ports better shoes sprites from Eris since my last drip PR for some reason didn't save those
- Removes legacy chem vomitol from most places where applicable, as it doesn't exist anymore
- Removes bike helms from random helmet chance at the complain of supply spec complaints about silly bike helmets cluttering their armory shop, if you want to spawn them you'll have to use/map in their pre-existing pool
- Minor typo fixes all around

<hr>

## Changelog
:cl:
add: Added Nanobot Fluid, a chem that can be used on humans with prosthetics to fix their external and internal component damage. Does not work on FBP's. Mix equal parts Cronexidone, Aluminum, Iron, with 1u of Raw Uncapped Nanites, in the prescence of 5u of Plasma, between 73 and 98 kelvin.
del: Removed extant references to vomitol
tweak: Kyphotorin now properly works on the dead, tried and tested. This only works on a cryotube and not by injecting a corpse with it, mind you.
balance: Cronexidone is now ACTUALLY superior to Cryoxadone instead of lying (buffs brute and burn heal amounts)
fix: Clothing, armor and RIG modules should no longer be able to take gunmods/laser guides
imageadd: Properly ports beter shoes sprites from Eris, since they didn't get properly ported on the drip PR
spellcheck: Fixed a few typos across the board, removing all references to "clone" damage replaced with "cellular corruption" as cloning is lore-wise not possible (besides vatborns)
refactor: Refactors "clonexadone" into "cronexidone" proper
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know-->
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/mods/mods.dm [hunk 1]
```diff
@@ -831,7 +831,7 @@
 	I.removal_time *= rand(10, 14)/10
 	I.removal_difficulty *= rand(5, 15)/10
 	I.gun_loc_tag = GUN_MUZZLE
-	I.prefix = "silenced"
+	I.prefix = "badly silenced"
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/mods/mods.dm:834]:
```suggestion
	I.prefix = "silenced"
```
its meant to trick people into use it

File: code/modules/projectiles/guns/mods/mods.dm [hunk 2]
```diff
@@ -867,7 +867,7 @@
 	I.removal_time *= rand(10, 14)/10
 	I.removal_difficulty *= rand(5, 15)/10
 	I.gun_loc_tag = GUN_SCOPE
-	I.prefix = "scoped"
+	I.prefix = "badly zeroed"
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/mods/mods.dm:870]:
```suggestion
	I.prefix = "scope"
```

File: code/modules/projectiles/guns/mods/mods.dm [hunk 3]
```diff
@@ -884,7 +884,7 @@
 	I.removal_time *= rand(10, 14)/10
 	I.removal_difficulty *= rand(5, 15)/10
 	I.gun_loc_tag = GUN_MECHANISM
-	I.prefix = "feather trigger"
+	I.prefix = "trigger feathered"
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/mods/mods.dm:887]:
```suggestion
	I.prefix = "feather trigger"
```

File: code/modules/projectiles/guns/mods/mods.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/modules/projectiles/guns/mods/mods.dm:834]: Eh, at this point everyone knows that no (good) weapon comes pre-modded but sure

File: code/modules/projectiles/guns/mods/mods.dm (hunk 3 above)
Comment (ThePainkiller) [reply code/modules/projectiles/guns/mods/mods.dm:887]: This one is actually a typo fix, it makes more grammatical sense this way

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/mods/mods.dm:887]: mind fixing them all then?

**Pull request merged**